api/v1alpha1/modelregistry_webhook.go

```diff
@@ -61,6 +62,14 @@ func (r *ModelRegistry) Default() {
 	if len(r.Spec.Rest.Image) == 0 {
 		r.Spec.Rest.Image = config.GetStringConfigWithDefault(config.RestImage, config.DefaultRestImage)
 	}
+
+	// FIX default database configs that get set for some reason in Kind cluster
```

Comment:
Is this something would you like to fix in this PR, I am not that sure I got the issue here.

Reply:
Changed comment to Fixes...